nss-day-cohort-48/rare-server-boastful-bulldogs
# registration

Can now register new users. 

##steps to test: 

1. pull down branch hm-registration
2. start debugger/react
3. create new user using fields
4. test login
5. use SELECT to ensure data is in the Users table
